packages/ember-runtime/lib/mixins/array.js

```diff
@@ -60,6 +60,10 @@ export function objectAt(content, idx) {
   return typeof content.objectAt === 'function' ? content.objectAt(idx) : content[idx];
 }
 
+export function pushObject(arr, obj) {
```

Comment:
I was basing this on the work done here: 924bf0e4235d586903e0886db7f5c567a86aff00 - not sure if thats a good starting point. Let me know.